docs/docs/how-to/cloud/working-with-redirects-and-rewrites.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Working with Redirects and Rewrites"
+description: "Learn how to leverage redirects, rewrites, and reverse proxies in Gatsby Cloud"
+label: "cloud"
+---
+
+## What is a redirect?
+
+Redirects are settings in the network layer that allow you to route traffic from one url path to another with little to no disruption.
+
+For instance, while rebuilding your cooking blog, you might want to move all of your recipes from their old path of blog/recipes to a new path of recipes. To make sure that all the existing links to your recipes still work, you would need a way to redirect your users from blog/recipes/mouthwatering-lasagna to recipes/mouthwatering-lasagna. No one wants to lose access to such a, well, mouthwatering recipe.
+
+## ♻️ How to create redirects in Gatsby Hosting
+
+In order to use redirects, you must include the gatsby-plugin-gatsby-cloud in your project.
```

Comment:
Package names are fenced in code blocks

```suggestion
In order to use redirects, you must include `gatsby-plugin-gatsby-cloud` in your project.
```

Also, isn't this information outdated since we auto-install the plugin for everyone?